lib/logproto/logproto-text-client.c

```diff
@@ -119,15 +119,14 @@ static LogProtoStatus
 log_proto_text_client_post(LogProtoClient *s, LogMessage *logmsg, guchar *msg, gsize msg_len, gboolean *consumed)
 {
   LogProtoTextClient *self = (LogProtoTextClient *) s;
-  gint rc;
 
   /* try to flush already buffered data */
   *consumed = FALSE;
-  rc = log_proto_text_client_flush(s);
-  if (rc == LPS_ERROR)
+  const LogProtoStatus status = log_proto_text_client_flush(s);
```

Comment:
Why the `const` ?

Reply:
I like to give the compiler and the reader a hint, it won't change.